ntbs-service/DataAccess/ReferenceDataRepository.cs

```diff
@@ -354,5 +356,16 @@ public async Task<IList<CaseManagerTbService>> GetCaseManagerTbServicesByUsernam
             return await _context.CaseManagerTbService.Where(cmtb => cmtb.CaseManager.Username == username)
                 .ToListAsync();
         }
+
+        public async Task<IList<User>> GetUsersByPhecAdGroup(string phecAdGroup)
+        {
+            return await _context.User.Where(u => u.AdGroups.Contains(phecAdGroup)).ToListAsync();
```

Comment:
According to the ticket "If you belong to any TB service AD group you are treated as a TB service user only." Are we enforcing that anywhere? Or are we deliberately moving to allow case managers to also be regional staff?

Reply:
I think it is enforced in the permissions model (here : https://github.com/publichealthengland/ntbs_Beta/blob/4ed59615c2a871662c63d71b14199b87f47ec20b/ntbs-service/Services/AuthorizationService.cs#L178).
Nancy has confirmed we are making the conscious decision to show everyone with the AD group membership, same for showing a user's regions in search results/profile.